shadows/framework/src/main/java/org/robolectric/shadows/ShadowContextHubManager.java

```diff
@@ -146,7 +151,9 @@ protected List<ContextHubInfo> getContextHubs() {
 
   @Nullable
   public List<ContextHubClient> getClientsWithAttributionTag(String attributionTag) {
-    return ImmutableList.copyOf(attributionTagToClientMap.get(attributionTag));
+    synchronized (attributionTagToClientMap) {
```

Comment:
> Why adding synchronized here? And I don't recommend to use a reference of a map modified by other code as synchronized' object. If you need, you can create a single pure object as synchronization object.

Fixed it and adding emulator test is not feasible for ContextHubManager because it is hidden